packages/website/pages/docs/resources/taiko-glossary.md

```diff
@@ -0,0 +1,95 @@
+
+## Decentralization
+
+The proposing and proving design where no single party is able to control all transaction ordering or is solely responsible for proving blocks. Being sufficiently decentralized implies that the protocol should keep working in a reliable manner in adversarial situations.
+
+## Deterministic
+
+The block execution is deterministic after the block is appended to the proposed block chain, all block properties are immutable and the post-block state can be calculated by anyone.
+
+## EtherVault
+
+A contract that holds some huge amount of ETHs on L2 and is able to authorize third party bridges to receive Ether from the EtherVault to execute deposit operations.
+
+## Fork Choice
+
+Because blocks can be proven in parallel, the Taiko smart contract can receive a proof for a block for which it doesn’t know the correct pre-state for. The smart contract thus accepts all valid proofs using different pre-states, once the smart contract know the correct pre-state as proofs for all parent blocks come in, the correct pre-state can be selected together with its corresponding post-state to advance the verified chain.
+
+## Golden Touch Address
+
+An address with a revealed private key to transact all [anchor transactions](#Anchor-Transaction).
+
+## Incentive multipliers
+
+Time sensitive multipliers that can decrease fees and increase rewards to incentivize proposals and proofs when there are unpredictable deterrents acting against the engagement of proposers or provers.
+
+## Intrinsic Validity Function
+
+A function that is run on each proposed block to determine if the block contains valid data. If the block doesn’t contain valid data the block will be seen as a block with an empty transaction list.
+
+## Invalid Block
+
+A block that fails to pass the [Intrinsic Validity Function](#Intrinsic-Validity-Function).
+
+## Metadata (of the block)
+
+A tuple of 9 items comprising: block id, beneficiary (20-byte address), gas limit, timestamp, mixHash value, extraData value, Keccak-256 hash, the enclosing L1 block’s parent block number, the enclosing L1 block’s parent block hash.
+
+## Verified block (on-chain verified)
+
+A block is on-chain verified when its proof is submitted and its parent block is verified.
+
+## Parallel Proof
+
+Blocks can be proven in parallel because all intermediate states between blocks are known and [deterministic](#Deterministic).
+
+## Proposing a block
+
+Any willing entity can propose new Taiko blocks using the [TaikoL1 contract](#TaikoL1-Contract). Blocks are appended to a list in the order they come in (which is dictated by Ethereum L1 validators/block builders). Once the block is in the list it is [verified](#Verified-block-(on-chain verified)) and nodes can apply its state to the latest L2 state. A proposed block in Taiko is the collection of information, [Metadata](#Metadata), and a list of transactions.
+
+## RLP decodable
```

Comment:
we should remove this in my opinion, because it is generic enough to google and not needed to specify in the taiko glossary